Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -129,43 +116,32 @@ - (instancetype)initWithLogsDirectory:(NSString *)logsDirectory
 
 #endif
 
-- (void)dealloc {
-    // try-catch because the observer might be removed or never added. In this case, removeObserver throws an exception
-    @try {
-        [self removeObserver:self forKeyPath:NSStringFromSelector(@selector(maximumNumberOfLogFiles))];
-        [self removeObserver:self forKeyPath:NSStringFromSelector(@selector(logFilesDiskQuota))];
-    } @catch (NSException *exception) {
-    }
+- (void)_deleteOldLogFiles {
```

Comment:
I think the name should contain the reason why we don't use `-deleteOldFiles` directly. Something like `deleteOldFilesForConfigurationChange` or even `configurationChanged` would make it more clear what the method does.

Reply:
You are right.